lab3/README.md

```diff
@@ -0,0 +1,208 @@
+```
+
+Refer [Create Subscriber via Webconsole](https://free5gc.org/guide/Webconsole/Create-Subscriber-via-webconsole/#5-add-new-subscriber) to create subscriber 
+
+And, attach to ueransim container and run ue
+```sh
+// attach to container
+docker exec -it ueransim bash
+
+// run ue
+./nr-ue -c config/uecfg.yaml
+```
+And the expected result looks like:
+```sh
+2024-07-05 12:29:35.111] [nas] [info] UE switches to state [MM-DEREGISTERED/PLMN-SEARCH]
+[2024-07-05 12:29:35.111] [rrc] [debug] New signal detected for cell[1], total [1] cells in coverage
+[2024-07-05 12:29:36.572] [nas] [info] Selected plmn[208/93]
+[2024-07-05 12:29:36.573] [rrc] [info] Selected cell plmn[208/93] tac[1] category[SUITABLE]
+[2024-07-05 12:29:36.573] [nas] [info] UE switches to state [MM-DEREGISTERED/PS]
+[2024-07-05 12:29:36.573] [nas] [info] UE switches to state [MM-DEREGISTERED/NORMAL-SERVICE]
+[2024-07-05 12:29:36.574] [nas] [debug] Initial registration required due to [MM-DEREG-NORMAL-SERVICE]
+[2024-07-05 12:29:36.575] [nas] [debug] UAC access attempt is allowed for identity[0], category[MO_sig]
+[2024-07-05 12:29:36.576] [nas] [debug] Sending Initial Registration
+[2024-07-05 12:29:36.576] [rrc] [debug] Sending RRC Setup Request
+[2024-07-05 12:29:36.579] [rrc] [info] RRC connection established
+[2024-07-05 12:29:36.592] [rrc] [info] UE switches to state [RRC-CONNECTED]
+[2024-07-05 12:29:36.593] [nas] [info] UE switches to state [MM-REGISTER-INITIATED]
+[2024-07-05 12:29:36.596] [nas] [info] UE switches to state [CM-CONNECTED]
+[2024-07-05 12:29:36.691] [nas] [debug] Authentication Request received
+[2024-07-05 12:29:36.692] [nas] [debug] Received SQN [000000000027]
+[2024-07-05 12:29:36.692] [nas] [debug] SQN-MS [000000000000]
+[2024-07-05 12:29:36.735] [nas] [debug] Security Mode Command received
+[2024-07-05 12:29:36.735] [nas] [debug] Selected integrity[2] ciphering[0]
+[2024-07-05 12:29:36.961] [nas] [debug] Registration accept received
+[2024-07-05 12:29:36.962] [nas] [info] UE switches to state [MM-REGISTERED/NORMAL-SERVICE]
+[2024-07-05 12:29:36.962] [nas] [debug] Sending Registration Complete
+[2024-07-05 12:29:36.962] [nas] [info] Initial Registration is successful
+[2024-07-05 12:29:36.963] [nas] [debug] Sending PDU Session Establishment Request
+[2024-07-05 12:29:36.964] [nas] [debug] UAC access attempt is allowed for identity[0], category[MO_sig]
+[2024-07-05 12:29:36.964] [nas] [debug] Sending PDU Session Establishment Request
+[2024-07-05 12:29:36.964] [nas] [debug] UAC access attempt is allowed for identity[0], category[MO_sig]
+[2024-07-05 12:29:37.173] [nas] [debug] Configuration Update Command received
+[2024-07-05 12:29:37.645] [nas] [debug] PDU Session Establishment Accept received
+[2024-07-05 12:29:37.645] [nas] [info] PDU Session establishment is successful PSI[1]
+[2024-07-05 12:29:37.688] [nas] [debug] PDU Session Establishment Accept received
+[2024-07-05 12:29:37.691] [nas] [info] PDU Session establishment is successful PSI[2]
```

Comment:
I suggest putting a screenshot, and must include the IP address allocation message.